src/app/Library/CrudPanel/CrudPanel.php

```diff
@@ -144,6 +144,28 @@ private function driverIsMongoDb()
         return $this->getSchema()->getConnection()->getConfig()['driver'] === 'mongodb';
     }
 
+    /**
+     * Check if the database connection is any sql driver.
+     *
+     * @return bool
+     */
+    private function driverIsSql()
+    {
+        $driver = $this->getSchema()->getConnection()->getConfig()['driver'];
+
+        return in_array($driver, $this->getSqlDriverList);
+    }
+
+    /**
+     * Get SQL driver list.
+     *
+     * @return array
+     */
+    private function getSqlDriverList()
```

Comment:
```suggestion
    public function getSqlDriverList()
```
This method should be public to be accessible in `HasRelationshipFields.php`.

Probably `driverIsSql()` and `driverIsMongoDb()` should be too.

Reply:
I think it's ok to have only this list getter public. Other two methods (mongo one would never be called is just there for BC, I added a deprecation notice to it).

`driverIsSql()` is called in CrudPanel context, so it's ok to be private.